<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Login</class>
 <widget class="QMainWindow" name="Login">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>714</width>
    <height>555</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>714</width>
    <height>555</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>714</width>
    <height>555</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Welcome</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resources.qrc">
    <normaloff>:/new/Images/member1.ico</normaloff>
    <normalon>:/new/Images/member1.ico</normalon>
    <activeon>:/new/Images/member1.ico</activeon>:/new/Images/member1.ico</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:white;</string>
  </property>
  <property name="tabShape">
   <enum>QTabWidget::Triangular</enum>
  </property>
  <property name="dockNestingEnabled">
   <bool>false</bool>
  </property>
  <property name="unifiedTitleAndToolBarOnMac">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>121</width>
      <height>131</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resources.qrc">:/new/Images/member2.png</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>0</y>
      <width>591</width>
      <height>131</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Koshgarian-Ligh</family>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:red;
background-color:black;</string>
    </property>
    <property name="text">
     <string> DEENZE FINANCE MANAGEMENT SYSTEM</string>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget_Welcome">
    <property name="geometry">
     <rect>
      <x>-14</x>
      <y>139</y>
      <width>731</width>
      <height>361</height>
     </rect>
    </property>
    <property name="baseSize">
     <size>
      <width>0</width>
      <height>0</height>
     </size>
    </property>
    <property name="palette">
     <palette>
      <active>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </active>
      <inactive>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </inactive>
      <disabled>
       <colorrole role="Button">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Base">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
       <colorrole role="Window">
        <brush brushstyle="SolidPattern">
         <color alpha="255">
          <red>255</red>
          <green>255</green>
          <blue>255</blue>
         </color>
        </brush>
       </colorrole>
      </disabled>
     </palette>
    </property>
    <property name="font">
     <font>
      <family>Trajan-Normal Wd</family>
      <pointsize>8</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="layoutDirection">
     <enum>Qt::RightToLeft</enum>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 18, 10);</string>
    </property>
    <property name="tabPosition">
     <enum>QTabWidget::North</enum>
    </property>
    <property name="tabShape">
     <enum>QTabWidget::Triangular</enum>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>16</height>
     </size>
    </property>
    <property name="elideMode">
     <enum>Qt::ElideMiddle</enum>
    </property>
    <property name="usesScrollButtons">
     <bool>true</bool>
    </property>
    <property name="tabsClosable">
     <bool>false</bool>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/new/Images/about.ico</normaloff>
       <normalon>:/new/Images/about.ico</normalon>
       <activeon>:/new/Images/about.ico</activeon>
       <selectedon>:/new/Images/about.ico</selectedon>:/new/Images/about.ico</iconset>
     </attribute>
     <attribute name="title">
      <string>About Deenze</string>
     </attribute>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>40</y>
        <width>521</width>
        <height>211</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro Bold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Deenze Finance Management System&lt;br&gt;Ver 1.0&lt;br&gt;Software used by SACCOS to manage their savings&lt;br&gt;Developed by Denson Ngumo&lt;br&gt;Warning: This computer program is protected by copyright law &lt;br&gt;and international treaties. &lt;br&gt;Copyright(c) 2014 Deenze Incorporated. All rights reserved.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>60</y>
        <width>141</width>
        <height>171</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="Resources.qrc">:/new/Images/member1.png</pixmap>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>20</y>
        <width>481</width>
        <height>20</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/new/Images/settings.ico</normaloff>
       <normalon>:/new/Images/settings.ico</normalon>
       <activeon>:/new/Images/settings.ico</activeon>:/new/Images/settings.ico</iconset>
     </attribute>
     <attribute name="title">
      <string>Settings</string>
     </attribute>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>101</x>
        <y>61</y>
        <width>153</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro Bold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Set Interest Rate(%):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>101</x>
        <y>94</y>
        <width>361</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro Bold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Set Minimum Amount of Deposit allowed(Kshs):</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>101</x>
        <y>127</y>
        <width>288</width>
        <height>54</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro Bold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Set Minimum Number of consecutive
deposits in a year required to take a loan:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Rate">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>60</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Amt">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>90</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>6</number>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_NumberOfDeposits">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>150</y>
        <width>51</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>3</number>
      </property>
     </widget>
     <widget class="Line" name="line_3">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>0</y>
        <width>20</width>
        <height>301</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_4">
      <property name="geometry">
       <rect>
        <x>640</x>
        <y>10</y>
        <width>20</width>
        <height>301</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>101</x>
        <y>187</y>
        <width>350</width>
        <height>27</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro Bold</family>
        <pointsize>12</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Set Time Period for Loan Payment( In Months):</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_Time">
      <property name="geometry">
       <rect>
        <x>500</x>
        <y>190</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="maxLength">
       <number>2</number>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>240</x>
        <y>10</y>
        <width>321</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro Bold</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
        <underline>true</underline>
       </font>
      </property>
      <property name="text">
       <string>Set the SACCO's Management Settings</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton_3">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>240</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color:black;
color:red;</string>
      </property>
      <property name="text">
       <string>Set SACCO Loan Limits</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton_Save">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>290</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;
background-color:black;</string>
      </property>
      <property name="text">
       <string>Save Settings</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton_Employees">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>240</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;
background-color:black;</string>
      </property>
      <property name="text">
       <string>Set Employees(Users)</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="icon">
      <iconset resource="Resources.qrc">
       <normaloff>:/new/Images/member1.ico</normaloff>
       <normalon>:/new/Images/member1.ico</normalon>
       <activeon>:/new/Images/member1.ico</activeon>:/new/Images/member1.ico</iconset>
     </attribute>
     <attribute name="title">
      <string>Welcome</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>0</y>
        <width>691</width>
        <height>141</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Adobe Caslon Pro Bold</family>
        <pointsize>11</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;justify&quot;&gt;&lt;span style=&quot; font-size:14pt; font-color:black;&quot;&gt;Welcome to Deenze Finance Management System!&lt;br&gt; An easy to use,efficient system for SACCOS and chamas which will allow them to easily&lt;br&gt; control the management of how they save money and take loans.&lt;br&gt;You are required to login to proceed&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>160</y>
        <width>172</width>
        <height>41</height>
       </rect>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(0, 0, 0);
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>Login</string>
      </property>
     </widget>
     <widget class="QCommandLinkButton" name="commandLinkButton_Exit">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>200</y>
        <width>121</width>
        <height>41</height>
       </rect>
      </property>
      <property name="layoutDirection">
       <enum>Qt::RightToLeft</enum>
      </property>
      <property name="styleSheet">
       <string notr="true">color:red;
background-color:black;</string>
      </property>
      <property name="text">
       <string>Exit</string>
      </property>
      <property name="checkable">
       <bool>false</bool>
      </property>
      <property name="checked">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label_CurrDate">
      <property name="geometry">
       <rect>
        <x>520</x>
        <y>260</y>
        <width>191</width>
        <height>21</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>Trajan-Normal Wd</family>
        <pointsize>8</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>time</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>714</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
